src/GitHub.VisualStudio/GitHubPackage.cs

```diff
@@ -94,16 +89,6 @@ void LogVersionInformation()
         }
     }
 
-    [Export(typeof(IGitHubClient))]
-    public class GHClient : GitHubClient
-    {
-        [ImportingConstructor]
-        public GHClient(IProgram program)
-            : base(program.ProductHeader)
```

Comment:
Just FYI, the reason this was here is so that MEF imports requesting the client will pick it up with the correct ProductHeader value, which sets the user agent for the application on api requests. Since we don't have anything importing it anymore via MEF, it's fine to kill it.